harness/Makefile

```
@@ -0,0 +1,23 @@
+PROFILE     ?= release
+# ROOT = the workers/ parent dir (every worker is a sibling of harness/)
+ROOT        := $(abspath ..)
+WORKERS     := harness iii-directory session-manager context-manager \
+               provider-anthropic provider-openai shell web
+# add llm-router above to also build/link it (sibling, not a declared dep)
+CARGO_FLAG  := $(if $(filter release,$(PROFILE)),--release,)
+
+.DEFAULT_GOAL := harness
+.PHONY: harness clean
+
+harness:
```

checkmake (0.3.2)

[warning] Target body for "harness" exceeds allowed length of 5 lines (7).

(maxbodylength)